assemble/src/main/scripts/create-jshell.sh

```diff
@@ -68,9 +78,11 @@ function main() {
     echo '// Essential Hadoop API'
     echo 'import org.apache.hadoop.io.Text;'
     echo
-    echo '// Initialization code'
-    echo 'System.out.println("Preparing JShell for Apache Accumulo");'
+    echo '// Initialization Code'
+    echo 'System.out.println("Preparing JShell for Apache Accumulo \n");'
     echo
+    echo '// Accumulo Client Build'
+    addClientBuild "$scriptPath/accumulo-client-build.jsh"
```

Comment:
You don't need to read the file line by line. You can just do:

```suggestion
    cat "$scriptPath/accumulo-client-build.jsh" | grep -v '^#'
```
Or you can just use [tail to skip the license header](https://stackoverflow.com/a/604871/196405). Or, instead of having a separate script with a new license header, you could just append the code lines into here along side these other echo statements.

Reply:
With the helpful suggestions above, the client implementation has been reduced significantly. As a result, I believe we can forgo the separate file and just inline the implementation. If there are any other Accumulo features we want to include in JShell, we can always move all Java implementations to a separate file if needed.